quilt/tools/main.py

```diff
@@ -152,7 +154,10 @@ def check_hash(group, hashstr):
         func(**kwargs)
         return 0
     except command.CommandException as ex:
-        print(ex, file=sys.stderr)
+        if re.search(r'Requirements file quilt.yml not found', str(ex)):
```

Comment:
This is a pretty crude hack. How about removing this and just adding a suggestion to run `quilt install -h` in the exception message (when the requirements file isn't found) in command.py line: 785?